public/resources/functions/inqueue-utils.js

```diff
@@ -157,104 +143,85 @@ exports.handler = JWEValidator(async function (context, event, callback) {
       }
 
       //  main logic
-      async function deleteMain() {
-        res = await getTask(event.taskSid);
-        res = await updateTask(event.taskSid, res.attr);
-        let del = await deleteTranscription(event.transcriptSid);
-        let cal = await deleteRecord(event.recordingSid);
 
-        callback(null, resp.setBody(res));
-      }
-      deleteMain();
+      let result = await getTask(event.taskSid);
```

Comment:
better to call these different variables? 

```js
const task = await getTask(event.taskSid);
const updatedTask = await updateTask(event.taskSid, task.attr);
```

Reply:
Solved in e98d19b